actions.yaml

```diff
@@ -33,3 +33,12 @@ pre-upgrade-check:
 
 resume-upgrade:
   description: Resume a rolling upgrade after asserting successful upgrade of a new revision.
+
+create-backup:
+  description: todo
+
+list-backups:
+  description: todo
+
+restore:
+  description: todo
```

Comment:
**niptick** should we comment this out if it is still a placeholder? Otherwise people could already use the action possibly... anyhow, I understand that this is only to be used in the feature branch, so it feels a minor

Reply:
That's a good point. I added those placeholders action because I needed at least one for testing that we can write to the object storage. And I thought at the time "why not add the others since I am there", but I now realise that this was short-sighted